packages/oid4vci-holder/src/agent/OID4VCIHolderService.ts

```diff
@@ -261,14 +261,14 @@ export const getIdentifierOpts = async (args: GetIdentifierArgs): Promise<Manage
         },
       },
     })
-    if (created) {
-      await agentContext.agent.emit(OID4VCIHolderEvent.IDENTIFIER_CREATED, { result })
-    }
     identifier = await context.agent.identifierManagedGetByDid({
       identifier: result,
       keyType,
       offlineWhenNoDIDRegistered: result.did.startsWith('did:ebsi:'),
     })
+    if (created) {
```

Comment:
we were passing in a value called result. what happened to that?

Reply:
The went inidentifierManagedGetByDid
The mobile wallet expects a managed identifier now, not a string